geode/src/test/java/org/apache/calcite/adapter/geode/rel/GeodeZipsTest.java

```diff
@@ -186,6 +207,103 @@ public void testItemPredicate() {
             + "      GeodeFilter(condition=[>(ITEM($2, 0), 0)])\n"
             + "        GeodeTableScan(table=[[geode, zips]])\n");
   }
+
+  @Test
+  public void testWhereWithOrForStringField() {
+    String expectedQuery = "SELECT state AS state FROM /zips "
```

Comment:
Is there a test with a single element ? Just 
```sql
where state in ('NY')
```

What about edge cases like `IN ('')` , `IN (null)`, `IN (true)` ?

Reply:
Added test for single state and also empty null true values